src/ex3/mondayu-logger-todoapp.js

```diff
@@ -0,0 +1,82 @@
+import fs from "fs";
+
+const args = process.argv;
+const currentDirectory = process.cwd() + "\\";
+
+if (fs.existsSync(currentDirectory + "todo.txt") === false) {
+  let createStream = fs.createWriteStream("todo.txt");
+  createStream.end();
+}
+
+export const listFunction = () => {
+  let data = [];
+  const fileData = fs.readFileSync(currentDirectory + "todo.txt").toString();
+
+  data = fileData.split("\n");
+
+  let filterData = data.filter(function (value) {
+    return value !== "";
+  });
+
+  if (filterData.length === 0) {
+    console.log("There are no pending tasks");
+  }
+
+  for (let i = filterData.length - 1; i >= 0; i--) {
+    console.log(filterData.length - i + ". " + filterData[i]);
+  }
+};
+
+export const addFunction = (task) => {
```

Comment:
why do you need to `Function` suffix in each function?
```suggestion
export const add = (task) => {
```

or
```suggestion
export const addTodo = (task) => {
```